test/regress/regress-35.txt

```diff
@@ -0,0 +1,25 @@
+;;; TOOL: wat2wasm
+;;; ARGS: -r
+(module 
+  ;; (type (func (param i32)(param i32)(result i32))) ;; fails, even if present
+  (func $add (param i32)(param i32)(result i32)
+    get_local 0
+    get_local 1
+    i32.add
+    return
+  )
+  (table 1 anyfunc)
+  (elem (i32.const 0) $add)
+  (func $run
+    i32.const 4444
+    i32.const 5555
+    i32.const 0
+    ;;call_indirect (type 0) ;; fails
```

Comment:
Should we also update the `test/dump/relocations.txt` to include this relocation type?    Or is that part of the TODO?

Reply:
Yeah, this was just to keep it from crashing. Actually supporting it is more work.